pymc3/distributions/bart.py

```diff
@@ -63,22 +63,30 @@ def __init__(
             )
         self.m = m
         self.alpha = alpha
-        self.y_std = Y.std()
-
-        if scale is None:
-            self.leaf_scale = NormalSampler(sigma=None)
-        elif isinstance(scale, (int, float)):
-            self.leaf_scale = NormalSampler(sigma=Y.std() / self.m ** scale)
 
         if inv_link is None:
-            self.inv_link = lambda x: x
+            self.inv_link = self.link = lambda x: x
+        elif isinstance(inv_link, str):
```

Comment:
Perhaps raise a ValueError if the string is not valid? Or somehow catch a None link/invlink at the end?